README.md

```diff
@@ -316,7 +313,7 @@ To consume a script package all we need to do specify the NuGet package in the `
 The following example loads the [simple-targets](https://www.nuget.org/packages/simple-targets-csx) package that contains script files to be included in our script.
 
 ```C#
-#! "netcoreapp2.1"
+#! "netcoreapp3.1"
```

Comment:
This is not used anymore so it should probably be removed from the docs

https://github.com/filipw/dotnet-script/commit/a10f53a99c8be51a0dcf5d3d7b0461d7d339649f

Reply:
thanks I forgot about that

Reply:
👍